### PR TITLE
ci: Get integration tests working and running again

### PR DESCRIPTION
**Pull request by dannygoldstein**
This PR gets integration tests working again, which have been not running since #5, testing against all the dashboard changes that have been made since then. 

Changes: 

1. Automates dev mode testing (you can now easily run integration tests against your local branch without having to rebuild assets). 
2. Uses supservisord as the process manager for integration tests.
3. Uses a library cypress-real-events to provide native system events that allow us to mock realistic hover events for showing hidden EEs. 
4. Updates dashboard.cy.ts to test the changes that have been made to dashboards since integration tests last ran.

### CODE REVIEW

File: integration_test/README.md [hunk 1]
```diff
@@ -1,13 +1,21 @@
-# Running in devmod
+# Running in devmode
 
-You want to run the frontend vite server and the weave server so you can develop. Here's how.
+Start the weave server and frontend dev server with 
+cd .. && make integration
 
-change baseUrl to 'http://localhost:3000' in cypress.config.js
+Start cypress with 
+yarn dev
```
Comment (tssweeney) [inline integration_test/README.md:7]: nit: cd ../weave-js

Comment (tssweeney) [inline integration_test/README.md:7]: nit: i had to `yarn install` first

**Review by tssweeney — APPROVED** [review]
SHIIP! I just tried this locally and it is awesome (: So much love for this!

File: integration_test/README.md (hunk 1 above)
Comment (dannygoldstein) [reply integration_test/README.md]: fixed

**Pull request merged**